index.html

```diff
@@ -212,7 +223,7 @@
 
 
       getFile("config.json", false, function(configData) {
-        config = JSON.parse(configData)
+        config = JSON.parse(JSON.minify(configData));
```

Comment:
I think it's useless to minimize data after it's been downloaded and before the parsing